src/whatsnew.adoc

```diff
@@ -95,95 +102,89 @@ their position in the table (i.e., the `$id` syntax).
+reader to determine the complete size of the encoding without having to inspect each byte individually.
+Finally, `FlexInt` does not use a separate bit to indicate its value's sign. Instead, it uses two's complement
+representation, allowing it to share much of the same structure and parsing logic as its unsigned counterpart.
+Benchmarks have shown that in aggregate, these encoding changes are between 1.25 and 3x faster than Ion 1.0's
+`VarUInt` and `VarInt` encodings depending on the host architecture.
+
+Ion 1.1 supplants link:https://amazon-ion.github.io/ion-docs/docs/binary.html#uint-and-int-fields[Ion 1.0's `Int`
+encoding primitive] with a new encoding called link:binary-encoding.adoc#fixedint[`FixedInt`], which uses two's
+complement notation instead of sign-and-magnitude.
+
+A new primitive encoding type, link:binary-encoding.adoc#flexsym[`FlexSym`], has been introduced to flexibly encode
+symbol IDs and symbolic tokens with inline text.
 
 ==== Type Encoding Changes
 
 All Ion types use the new low-level encodings as specified in the previous section.  Many of the opcodes used in Ion 1.0
 have been re-organized primarily to make E-expressions compact.
 
-Typed `null` values are now all encoded with two bytes with the `0xAE` opcode.  Lists and S-expressions have two
-encodings, the Ion 1.0 length prefixed encoding and the new delimited form that end with the `0xAD` end opcode.  Struct
-values have three encodings, the Ion 1.0 length prefixed encoding, the length prefixed encoding using `VarSym` to encode
-field names (allowing for inline symbol text), and the delimited form that encodes its field names with `VarSym` (there
-is no delimited form with Ion 1.0 encoding of field names).  Symbol values have two encodings, one is the Ion 1.0
-encoding with the symbol ID, and the other one is structurally identical to the encoding of strings, which are used for
-symbols with inline text.
+Typed `null` values are now link:binary-encoding.adoc#nulls[encoded in two bytes using the `0xEB` opcode].
 
-Annotated values no longer have an outer length container.  They are now encoded with an opcode that specifies a single
-annotation with value following; an opcode that specifies two annotations with a value following; and finally, an opcode
-that specifies a variable length of annotations followed by a value.  The latter encoding is similar to how Ion 1.0
-annotations are encoded with the exception that there is no outer length.
+Lists and S-expressions have link:binary-encoding.adoc#lists[two encodings]: a length-prefixed encoding and a new
+delimited form that ends with
+the `0xF0` opcode.
+
+Struct values have link:binary-encoding.adoc#structs[three encodings]: a length-prefixed encoding which uses symbol IDs
+for its field names, a length-prefixed encoding which uses `FlexSym` for its field names (allowing for inline symbol text
+as needed), and a delimited form that encodes its field names with `FlexSym` (there is no delimited form with symbol ID
```

Comment:
In the sentence about delimited lists and S-expressions we explicitly called out that they end with the `0xF0` opcode. Should we mention that here too for consistency?

Reply:
Done. I spelled out that it ends with a `VarSym` escape (`0x00`) followed by the `0xF0` opcode.